packages/thirdweb/src/transaction/prepare-contract-call.ts

```diff
@@ -52,6 +54,7 @@ export type PrepareContractCallOptions<
   > & {
     contract: ThirdwebContract<TAbi>;
     method: TMethod | TPreparedMethod;
+    extraCallData?: Hex;
```

Comment:
can you add this in `StaticPrepareTransactionOptions` instead? that way its easy to make it work with extension functions too (which you need to handle as well)

Reply:
Done